testdata/Dockerfile

```diff
@@ -26,10 +26,10 @@ RUN curl https://www.postgresql.org/media/keys/ACCC4CF8.asc | apt-key add - \
     && apt-get update \
     && env ACCEPT_EULA=Y apt-get install -y git postgresql-client-9.6 mysql-client-5.7 mssql-tools unixodbc-dev
 
-# Install Go
-RUN curl -o $GODIST https://storage.googleapis.com/golang/$GODIST \
+# Install Go + Go based tooling
+ENV GOLANG_VERSION 1.10
+RUN curl -o go.tar.gz https://storage.googleapis.com/golang/go$GOLANG_VERSION.linux-amd64.tar.gz \
```

Comment:
As a matter of principle I think this line should be written as:
```
RUN curl -o go.tar.gz "https://storage.googleapis.com/golang/go${GOLANG_VERSION}.linux-amd64.tar.gz" \
```

I'm not a fan of undelimited replacements in shell scripts, because it leaves it up to wishy washy rules and it's somewhat unclear to read.